<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>934</width>
      <height>472</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>2</number>
     </property>
     <item row="0" column="8">
      <widget class="QDoubleSpinBox" name="tSpinBox">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000500000000000</double>
       </property>
       <property name="value">
        <double>0.012500000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QDoubleSpinBox" name="alphaSpinBox">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="label_4">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>alpha</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_t">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="12">
      <widget class="QSpinBox" name="maxIterationSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max Iteration</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Iteration</string>
       </property>
      </widget>
     </item>
     <item row="0" column="13">
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Number of  Sites</string>
       </property>
      </widget>
     </item>
     <item row="0" column="14">
      <widget class="QSpinBox" name="numberOfSitesSpinBox">
       <property name="minimum">
        <number>64</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
       <property name="singleStep">
        <number>64</number>
       </property>
       <property name="value">
        <number>256</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="8">
      <widget class="QLabel" name="status_label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="1" column="2" colspan="13">
      <layout class="QHBoxLayout" name="spaceGrid"/>
     </item>
     <item row="2" column="11" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Initial Function</string>
       </property>
      </widget>
     </item>
     <item row="2" column="13" colspan="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Quadratic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Piecewise Linear 0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Piecewise Linear 0.2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non Continuos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gausian</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>tSpinBox</tabstop>
  <tabstop>pauseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
